pkgs/applications/graphics/ImageMagick/6.x.nix

```diff
@@ -109,10 +110,13 @@ stdenv.mkDerivation rec {
     done
   '';
 
+  passthru.tests.pkg-config = testers.testMetaPkgConfig finalAttrs.finalPackage;
```

Comment:
Do we really need to copy paste this to every package?

Reply:
We want ofborg to validate it so the alternative would be adding to `make-derivation.nix`.

Reply:
How bad would the impact be when it is unused?
IMO we definitely will forget this in some packages.

Reply:
https://github.com/NixOS/nixpkgs/issues/215162 tracking issue for this nuisance.